<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotCalibrationClass</class>
 <widget class="QMainWindow" name="RobotCalibrationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RobotCalibration</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addRefButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>add reference</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="robotCalibrationButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>robotCalibration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trackButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>tracking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="caliButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>caliRefState</string>
    </property>
   </widget>
   <widget class="QPushButton" name="probeButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>probeRefState</string>
    </property>
   </widget>
   <widget class="QPushButton" name="robotButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>robotRefState</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toolCalibrationButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>toolCalibration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibratorButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>calibratorState</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RobotCalibration.qrc"/>
 </resources>
 <connections/>
</ui>
